platforms/hyperledger-besu/configuration/reset-network.yaml

```diff
@@ -110,7 +110,72 @@
         - "{{ network['organizations'] }}"
       ignore_errors: yes
 
-    ##Delete Flux namespace 
+    # Delete ClusterRoleList related to flux
```

Comment:
Please change it to `Delete clusterrole ...`

Reply:
Hi @jagpreetsinghsasan 

Updated to "# Delete ClusterRole related to flux" in the latest Pull Request. Thank you.